packages/forklift-console-plugin/src/modules/Plans/views/create/PlanCreatePage.tsx

```diff
@@ -59,65 +58,61 @@ export const PlanCreatePage: React.FC<{ namespace: string }> = ({ namespace }) =
-          projectName={projectName}
-          filterState={filterState}
-          filterDispatch={filterDispatch}
-          dispatch={dispatch}
-          state={state}
-          providers={providers}
-          selectedProvider={selectedProvider}
-        />
-      ),
-      enableNext: filterState?.selectedVMs?.length > 0,
-    },
-    {
-      id: 'step-2',
-      name: 'Create migration plan',
-      component: (
-        <ProvidersCreateVmMigrationPage
-          state={state}
-          dispatch={dispatch}
-          emptyContext={emptyContext}
-        />
-      ),
-      enableNext:
-        !emptyContext &&
-        !(
-          !!state?.flow?.apiError ||
-          Object.values(state?.validation || []).some((validation) => validation === 'error') ||
-          state?.validation?.planName === 'default'
-        ),
-      canJumpTo: filterState?.selectedVMs?.length > 0,
-      nextButtonText: 'Create migration plan',
-    },
-  ];
-
   const title = 'Plans wizard';
   return (
     <>
       <PageSection variant="light">
         <Title headingLevel="h2">{'Create migration plan'}</Title>
       </PageSection>
-
-      <PageSection variant="light">
+      <PageSection variant="light" className="forklift--create-plan--wizard-container">
         <Wizard
-          className="forklift--create-plan--wizard-appearance-order"
-          navAriaLabel={`${title} steps`}
-          mainAriaLabel={`${title} content`}
-          steps={steps}
+          className="forklift--create-plan--wizard-content"
+          shouldFocusContent
+          title={title}
+          startIndex={startAtStep}
+          onClose={() => history.goBack()}
           onSave={() => {
             setActiveNamespace(state.underConstruction.projectName);
             dispatch(startCreate());
           }}
-          onClose={() => history.goBack()}
-          startAtStep={startAtStep}
-        />
+        >
+          <WizardStep
+            name="Select source provider"
+            id="step-1"
+            footer={{ isNextDisabled: !filterState?.selectedVMs?.length }}
+          >
+            <SelectSourceProvider
+              projectName={projectName}
+              filterState={filterState}
+              filterDispatch={filterDispatch}
+              dispatch={dispatch}
+              state={state}
+              providers={providers}
+              selectedProvider={selectedProvider}
+            />
+          </WizardStep>
+          <WizardStep
+            name="Create migration plan"
+            id="step-2"
+            isDisabled={!filterState?.selectedVMs?.length}
+            footer={{
+              nextButtonText: 'Create migration plan',
+              isNextDisabled:
+                emptyContext ||
+                !!state?.flow?.apiError ||
+                Object.values(state?.validation || []).some(
+                  (validation) => validation === 'error',
+                ) ||
+                state?.validation?.planName === 'default',
```

Comment:
seems like state.validation can be object or array? 

also consider extracting it to a isDsiabled func

Reply:
`validation` is an object property within the [ `CreateVmMigrationPageState` ](https://github.com/kubev2v/forklift-console-plugin/blob/cc62d29ef2e11deae2cd3ca6ae6857b69ffa8222/packages/forklift-console-plugin/src/modules/Providers/views/migrate/types.ts#L32-L38) interface, this huge state object is very complicated to track and debug as is so I prefer to leave it and not extract it to a local env.

But I extract it to separate funcs to make it more readable.